classes/panzer.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+class Panzer extends Tank{
```

Comment:
I think you should change the of the panzer class because panzer in german means tank and we will have a confusion,what is your opinion?

Reply:
The more you know 😂 
Can you give me a proper name, please?

Reply:
LightTank

Reply:
Done